tests/bench/beam.py

```
@@ -311,269 +228,69 @@ async def wait_for_deriver_queue_empty(
-                "alignment": alignment,
-                "overall_score": (tau + 1) / 2,  # Normalize to [0, 1]
-                "overall_reasoning": f"Kendall tau-b coefficient: {tau:.3f}. Extracted {len(extracted_events)} events from response.",
-            }
-
+                if response.status_code != 204:
+                    print(
+                        f"[{workspace_id}] ERROR: Dream trigger failed with status {response.status_code}"
+                    )
+                    print(f"[{workspace_id}] Response body: {response.text}")
+                    return False
         except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)